include/stdexec/execution.hpp

```diff
@@ -688,6 +691,9 @@ namespace stdexec {
 
   /////////////////////////////////////////////////////////////////////////////
   // [execution.senders]
+  template <class _T>
+    using attrs_of_t = decltype(get_attrs(__declval<_T>()));
```

Comment:
I'll keep the attrs stuff in place until the R7 changes are "finalized." attrs->env seems to be a relatively "uninteresting" change.

Reply:
I think I'd rather make the changes in brycelelbach/wg21_p2300_execution#20 as soon as possible. I don't want folks to start writing to the `get_attrs` API, and I don't want to have to support it indefinitely.

Reply:
I can put up a attrs->env PR now if you'd like based on what is currently on that PR.

Reply:
Yes, along with the `enable_sender` / `enable_receiver` change. That change must be made together with the `get_attrs`->`get_env` change or else senders will start satisfying the `receiver` concept. :grimacing:

Reply:
> attrs->env seems to be a relatively "uninteresting" change.
 
 Turns out that was a wrong assessment :P